source/_patterns/00-base/_color-variants.scss

```diff
@@ -0,0 +1,27 @@
+@use "sass:list";
+@use "@db-ui/base/build/scss/color-placeholder" as *;
+
+$color-variants: "neutral-0", "neutral-1", "neutral-3", "neutral-4", "neutral-5",
+	"neutral-6", "primary", "primary-light", "secondary", "secondary-light",
+	"critical", "critical-light", "success", "success-light", "warning",
+	"warning-light", "information", "information-light";
+
+@mixin color-variants($ia) {
+	@each $variant in $color-variants {
+		&[data-color-variant="#{$variant}"] {
+			@extend %db-bg-#{$variant};
+		}
+		@if ($ia == true) {
+			&[data-color-variant="#{$variant}-ia"] {
+				@extend %db-bg-#{$variant}-ia;
+			}
+		}
+	}
+}
+
+%color-variants {
+	@include color-variants(false);
+}
+%color-variants-ia {
+	@include color-variants(true);
+}
```

Comment:
will this get moved to base or remain in core ?

Reply:
Good question, I was insecure about it.

I thought we just need this scss file only for components, so the it would be better in core.
But I could move it in base as well, what do you think?

Reply:
From my understanding it belongs to its brothers and sisters in base.

Reply:
https://github.com/db-ui/base/pull/165